docs/UserGuide.adoc

```diff
@@ -409,50 +410,84 @@ Format: `exit`
 TeachConnect data are saved in the hard disk automatically [even while sharing TeachConnect] after any command that changes the data. +
 There is no need to save manually.
 
-=== Importing the contacts : `import` `[since v1.2]`
+=== Importing the contacts : `import` `[since v1.4]`
 
 TeachConnect allows you to import contacts from a different TeachConnect file by specifying the location of the file. +
 
 Format: `import [TYPE] pathname`
 
 Examples:
-Importing Contacts : `import ./data/importsample.xml`
+* Importing Contacts : `import ./data/importsample.xml`
 
-=== Exporting the contacts : `export` `[since v1.2]`
+=== Exporting the contacts : `export` `[since v1.4]`
 
-TeachConnect allows you to export contacts from your TeachConnect by specifying the name of the file you want to save it in and the path where you want to save it. It can export the contacts based on a given range of indexes or a given tag or a given tag in a range of indexes.
+TeachConnect allows you to export contacts from your TeachConnect by specifying the name of the file you want to save it in and the path where you want to save it. It can export the contacts based on a given range of indexes or a given tag or a given tag in a range of indexes. By specifying the type of the export you want it saves either only the xml file or both the xml and Csv file.
 
```

Comment:
TeachConnect allows you....
-->
Exports contacts...
TeachConnect can export

--> consistency with other commands